kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -135,7 +135,7 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/ui', 'uti
         const required = ['job_id', 'status'];
         return !!(
             jobState !== null &&
-            typeof jobState === 'object' &&
+            Utils.objectToString(jobState) === 'Object' &&
```

Comment:
as suggested in the previous PR, but mistakenly committed the edit in this branch